### DISCUSSION TITLE
Request offset maximum 12400. Please help

### DISCUSSION THREAD

**Discussion by nhattq** [etsy/open-api#1030] [Q&A]
Request offset maximum 12400. Please help

**Comment by Gareth064** [comment]
Hi @nhattq 
Are you able to explain in more detail what your question is?

**Reply by nhattq** [reply]
Yes, i'm getting list shop by shop name with offset 12400 then error "error: offset maximum". how can i increase that offset

**Reply by Gareth064** [reply]
The offset maximum is 12000

Mentions it in the documentation here 

https://developers.etsy.com/documentation/essentials/urlsyntax/#pagination

**Reply by Gareth064** [reply]
This has been discussed here previously

https://github.com/etsy/open-api/discussions/952
